### PR TITLE
Added Serilog for logging

### PR DESCRIPTION
**Pull request by rukasakurai**

### CODE REVIEW

File: modules/WebApp/HttpModuleClient.cs [hunk 1]
```diff
@@ -70,21 +74,21 @@ public async Task<MessageResponse> PipeMessage(Message message, object userConte
...
                         if(productData.Products.Count() == 0){
-                            Console.WriteLine("Queuing 'Live' Feed for displaying in WebApp.");
+                            _logger.LogInformation("Queuing 'Live' Feed for displaying in WebApp.");
                             shelfQueue.QueueShelf(productData);
                         } 
                     }
                 // catch and swallow exceptions 
                 } catch (AggregateException ex){
-                    Console.WriteLine($"Error processing message: {ex.Flatten()}");
+                    _logger.LogInformation($"Error processing message: {ex.Flatten()}");
```
Comment (angellmethod) [inline modules/WebApp/HttpModuleClient.cs]: I think for error messages you would want to use LogError

File: modules/WebApp/HttpModuleClient.cs [hunk 2]
```diff
@@ -70,21 +74,21 @@ public async Task<MessageResponse> PipeMessage(Message message, object userConte
...
                 } catch (Exception ex){
-                    Console.WriteLine($"Error processing message: {ex}");
+                    _logger.LogInformation($"Error processing message: {ex}");
```
Comment (angellmethod) [inline modules/WebApp/HttpModuleClient.cs]: I think for error messages you would want to use LogError

File: modules/WebApp/InProcShelfDataGenerator.cs [hunk 3]
```diff
@@ -61,24 +66,24 @@ public async Task QueueShelf(Shelf productData)
...
                     if(productData != null) {
-                        Console.WriteLine("Shelf Data Generator queueing in progress...");
+                        _logger.LogInformation("Shelf Data Generator queueing in progress...");
                         shelfQueue.QueueShelf(productData);
                     }
             } 
             catch (AggregateException ex)
             {
-                Console.WriteLine($"Error processing message: {ex.Flatten()}");
+                _logger.LogInformation($"Error processing message: {ex.Flatten()}");
```
Comment (angellmethod) [inline modules/WebApp/InProcShelfDataGenerator.cs]: LogError?

File: modules/WebApp/InProcShelfDataGenerator.cs [hunk 4]
```diff
@@ -61,24 +66,24 @@ public async Task QueueShelf(Shelf productData)
...
             }
             catch (Exception ex)
             {
-                Console.WriteLine($"Error processing message: {ex}");
+                _logger.LogInformation($"Error processing message: {ex}");
```
Comment (angellmethod) [inline modules/WebApp/InProcShelfDataGenerator.cs]: LogError?

File: modules/WebApp/HttpModuleClient.cs (hunk 1 above)
Comment (rukasakurai) [reply modules/WebApp/HttpModuleClient.cs]: Changed log level to error as suggested

File: modules/WebApp/HttpModuleClient.cs (hunk 2 above)
Comment (rukasakurai) [reply modules/WebApp/HttpModuleClient.cs]: Changed log level to error as suggested

File: modules/WebApp/InProcShelfDataGenerator.cs (hunk 3 above)
Comment (rukasakurai) [reply modules/WebApp/InProcShelfDataGenerator.cs]: Changed log level to error as suggested

File: modules/WebApp/InProcShelfDataGenerator.cs (hunk 4 above)
Comment (rukasakurai) [reply modules/WebApp/InProcShelfDataGenerator.cs]: Changed log level to error as suggested

**Comment by rukasakurai** [comment]
@angellmethod Thanks for the comments. Changed log level to error as suggested.

**Pull request merged**